docs/_data/table/en/client-metrics.yml

```diff
@@ -1,6 +1,10 @@
-Client.BlockReadDataChunk:
+Client.BlockReadDataChunkDirect:
   'The timer statistics of reading block data in chunks from Alluxio workers. This metrics will only be recorded when alluxio.user.block.read.metrics.enabled is set to true'
-Client.BlockReadDataFromChunk:
+Client.BlockReadDataChunkRemote:
```

Comment:
@LuQQiu  I think this metrics `Client.BlockReadDataChunkRemote` is only introduced in https://github.com/Alluxio/alluxio/pull/13438/, which has not been merged , right?